control-plane/config/eventing-kafka-source/200-controller/500-controller.yaml

```diff
@@ -0,0 +1,118 @@
+---
+
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-controller
+  namespace: knative-eventing
+  labels:
+    app: kafka-controller
+    kafka.eventing.knative.dev/release: devel
```

Comment:
should we call this differently like `kafka-source-controller` and use a different label value?

Reply:
I was thinking leaving the same so it doesn't confuse anyone used to the existing ekb deployments and simply switching to use only the source...? Plus it still includes other controllers like consumergroup, consumer which are controllers used by other resources too. wdyt?

Reply:
Sounds good!

Reply:
however, what if we want to have a `KafkaSink` bundle calling it `kafka-controller` again, doesn't seem doable in that case.

Reply:
hmm if at the same time as KafkaSource bundle.
The post-install scripts and cluster role yamls etc have hardcoded dependence on `kafka-controller`....that ok to ignore this new controller?

Reply:
Yes, ideally, we have different cluster roles as well, but keeping them in sync maybe a burden